.circleci/config.yml

```
@@ -722,6 +722,22 @@ jobs:
         # too early without Saucelabs not being ready.
       - run: ./scripts/saucelabs/wait-for-tunnel.sh
       - run: yarn karma start ./karma-js.conf.js --single-run --browsers=${KARMA_JS_BROWSERS}
+        # run zone.js saucelabs test
+      - run: cd packages/zone.js
+      - run: yarn
+      - run: yarn add jasmine@2.9.0 jasmine-core@2.9.0
+      - run: yarn karma start karma-dist-sauce-jasmine.conf.js --single-run
+      - run: yarn karma start karma-build-sauce-mocha.conf.js --single-run
+      - run: yarn karma start karma-dist-sauce-selenium3-jasmine.conf.js --single-run
+      - run: yarn karma start karma-build-sauce-selenium3-mocha.conf.js --single-run
+      - run: yarn karma start karma-dist-sauce-jasmine3.conf.js --single-run --error-policy=disable
+      - run: yarn karma start karma-dist-sauce-jasmine3.conf.js --single-run --error-policy=lazy
+      - run: node simple-server.js 2>&1> server.log&
+      - run: node ./test/webdriver/test.sauce.js
+      - run: yarn add jasmine@3.0.0 jasmine-core@3.0.0 mocha@5.0.1
+      - run: yarn karma start karma-dist-sauce-jasmine3.conf.js --single-run
+      - run: yarn karma start karma-build-sauce-selenium3-mocha.conf.js --single-run
+      - run: yarn karma start  karma-evergreen-dist-sauce-jasmine.conf.js --single-run
```

```suggestion
      - run: yarn karma start karma-evergreen-dist-sauce-jasmine.conf.js --single-run
```
Also, since there are too many commands to test `zone.js` on soucelabs, could we move them into a script inside `packages/zone.js/`? It makes the CI config easier to follow.

---

So now since only `CI_REQUIRED` browsers need to be tested, I only left one test here.